src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,10 +1,82 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
+import { Movie } from '../../types/Movie';
 import { TextField } from '../TextField';
 
-export const NewMovie = () => {
+type Props = {
+  onAdd: (movie: Movie) => void,
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, IncreaseCount] = useState(0);
```

Comment:
Follow the convention of the `useState` callback
```suggestion
  const [count, setCount] = useState(0);
```